packages/firebase_messaging/firebase_messaging/example/lib/main.dart

```diff
@@ -4,6 +4,8 @@
 
 // @dart=2.9
 
+// ignore_for_file: avoid_print
```

Comment:
instead disable `avoid_print` for the example folder (can be done by adding another analysis_options.yaml in the example folder)